src/Point.php

```diff
@@ -328,4 +329,19 @@ public function getIterator()
     {
         return new \ArrayIterator($this->toArray());
     }
+
+    /**
+     * Calculates the azimuth of the subject relative to the observer (this point).
```

Comment:
Can you please add the full description that you added in `GeometryEngine::azimuth()`?

Reply:
Could you please elaborate what you mean here. Should it go like this?

> Calculation is performed by the `GeometryEngine::azimuth()`.

Reply:
No, I mean this description:

     * Returns the azimuth in radians of the segment defined by the given point geometries.
     * The azimuth is an angle measured from the north, and is positive clockwise:
     * North = 0; East = π/2; South = π; West = 3π/2.

Reply:
Fixed it.